src/rt/minfo.d

```diff
@@ -393,9 +393,6 @@ struct ModuleGroup
             ctoridx = 0;
             foreach (int idx, m; _modules)
             {
-                // TODO: Should null ModuleInfo be allowed?
-                if (m is null)
-                    continue;
```

Comment:
Wait, as @klickverbot said, it might still happen with LDC. Maybe just change to comment to mention LDC.